src/contracts/Protocol/Vault.sol

```diff
@@ -122,10 +123,15 @@ contract Vault is Ownable, OnlyCostController {
 
     /**
      * @notice Allows a user to withdraw METRIC locked for a question, after the question is published.
+     * @param user The address of the user withdrawing the METRIC
      * @param questionId The question id
      * @param stage The stage for which the user is withdrawing metric from a question.
      */
-    function withdrawMetric(uint256 questionId, STAGE stage) external {
+    function withdrawMetric(
+        address user,
+        uint256 questionId,
+        STAGE stage
+    ) external onlyApi {
         // Checks if Metric is withdrawn for a valid stage.
         if (uint8(stage) >= 5) revert InvalidStage();
```

Comment:
I know the change wasn't made in this one but 5 feels arbitrary why not > STAGE.REVIEW

Reply:
Good idea